reloc/build_debuginfo.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash -e
+
+. /etc/os-release
+print_usage() {
+    echo "build_debuginfo.sh --reloc-pkg build/release/scylla-package.tar.gz"
+    echo "  --reloc-pkg specify relocatable package path"
+    echo "  --builddir specify rpmbuild directory"
+    exit 1
+}
+RELOC_PKG=build/release/scylla-package.tar.gz
+BUILDDIR=build/debuginfo
+while [ $# -gt 0 ]; do
+    case "$1" in
+        "--reloc-pkg")
+            RELOC_PKG=$2
+            shift 2
+            ;;
+        "--builddir")
+            BUILDDIR="$2"
+            shift 2
+            ;;
+        *)
+            print_usage
+            ;;
+    esac
+done
+
+if [ ! -e $RELOC_PKG ]; then
+    echo "$RELOC_PKG does not exist. Please build it with 'ninja' before running this script."
+    exit 1
+fi
+RELOC_PKG=$(readlink -f $RELOC_PKG)
+rm -rf $BUILDDIR
+mkdir -p $BUILDDIR
+tar -C $BUILDDIR/ -xpf $RELOC_PKG scylla/SCYLLA-RELEASE-FILE scylla/SCYLLA-VERSION-FILE scylla/libexec scylla/node_exporter
+cd $BUILDDIR/scylla
+
+SCYLLA_VERSION=$(cat SCYLLA-VERSION-FILE)
+SCYLLA_RELEASE=$(cat SCYLLA-RELEASE-FILE)
+
+RPM_BUILD_ROOT=$(pwd) RPM_BUILD_DIR=$(pwd) /usr/lib/rpm/find-debuginfo.sh -j$(nproc) -n -m -i --unique-debug-suffix -$SCYLLA_VERSION-$SCYLLA_RELEASE.$(arch) $(pwd)
```

Comment:
It's sad to unpack and repack the package just to execute find-debuginfo. Maybe we can execute the commands directly after link from https://stackoverflow.com/questions/16408893/gnu-ld-gdb-separate-debug-files-how-to-produce-the-debug-file-when-theres-too.

Reply:
Okay, I will try